concent_api/core/migrations/0010_auto_20180311_2101.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
There's a lot of migrations in this branch. Maybe you could squash them all into a single one?

Reply:
I will when I squash fixup with commit which changes models.

Reply:
OK. As far as I'm concerned you can squash the fixups already. I still have one to review but I have it in my local working copy.

Reply:
Squashed